src/chainparams.cpp

```diff
@@ -426,10 +427,10 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_MN_RR].useEHF = true;
 
         // The best chain should have at least this much work.
-        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000000002ecd6cf5ad0f774"); // 960000
+        consensus.nMinimumChainWork = uint256S("0x000000000000000000000000000000000000000000000000031779704a0f54b4"); // 1069875
```

Comment:
assumevalid for testnet should be far in the past for case of any possible re-ogranization, like months behind.

If you still want to bump it for testnet, you may choose a block before MN_RR activation: [my branch](https://github.com/knst/dash/commit/0bcdfb98783698408b917ca762e807f83ebdbba6)

Reply:
It's impossible to reorg testnet other than via an intentional hard-fork because it's chainlocked and in that case you'd might want to update these numbers anyway 🤷‍♂️